platform/android/plugin/godot_plugin_jni.cpp

```diff
@@ -114,16 +114,19 @@ JNIEXPORT void JNICALL Java_org_godotengine_godot_plugin_GodotPlugin_nativeEmitS
 	String signal_name = jstring_to_string(j_signal_name, env);
 
 	int count = env->GetArrayLength(j_signal_params);
-	const Variant *args[count];
+	Variant args[VARIANT_ARG_MAX];
 
-	for (int i = 0; i < count; i++) {
+	ERR_FAIL_COND_MSG(count > VARIANT_ARG_MAX, "Maximum argument count exceeded!")
```

Comment:
This line is missing a `;`

Reply:
Good catch! Didn't notice that because it didn't lead to a compile error. Fixed :)

Reply:
It should lead to a compilation error normally, that's weird.